### PR TITLE
Automation of the artifact downloads

### PR DESCRIPTION
**Pull request by zelzhan**
…acts

### CODE REVIEW

**Review by thealphadollar — CHANGES_REQUESTED** [review]
@zelzhan I think we should have a better commit message, fewer file changes per commit, and better pull request descriptions. It gets really messy to review stuff otherwise and these are basic open-source etiquette :)

Please refer to below links for more information:

- https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/
- https://www.atlassian.com/blog/git/written-unwritten-guide-pull-requests
- https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices

In short, write more detailed yet straight-forward commit messages and make commits more frequent so it is easier to revert later and easier to review.

cc/ @canihavesomecoffee
